src/stdlib_experimental_kinds.f90

```diff
@@ -0,0 +1,10 @@
+module stdlib_experimental_kinds
+! Instead of iso_fortran_env, we use iso_c_binding, to be compatible with C
+!use iso_fortran_env, only: sp=>real32, dp=>real64, qp=>real128
+!use iso_fortran_env, only: int32, int64, int128
+use iso_c_binding, only: sp=>c_float, dp=>c_double, qp=>c_float128
+use iso_c_binding, only: int32=>c_int32_t, int64=>c_int64_t, int128=>c_int128_t
+implicit none
+private
+public sp, dp, qp, int32, int64, int128
```

Comment:
```
public sp, dp, qp, int8, int16, int32, int64
```

Reply:
@jvdp1 go ahead and make the suggestion in GitHub. (Select the line and click "make suggestion"). Then I can just click "commit".